frame/nomination-pools/src/lib.rs

```diff
@@ -2342,6 +2738,7 @@ impl<T: Config> Pallet<T> {
 			member: member_account.clone(),
 			pool_id: member.pool_id,
 			payout: pending_rewards,
+			commission: maybe_commission.as_ref().map(|(c, _)| *c).unwrap_or(Zero::zero()),
```

Comment:
```suggestion
			commission: maybe_commission.map(|(c, _)| *c).unwrap_or(Zero::zero()),
```

probably can be consumed as it is the last usage.

Reply:
The compiler doesn't like that as it is behind a shared reference.